### PR TITLE
Minor cleanups

### PR DESCRIPTION
**Pull request by astorije**

### CODE REVIEW

**Comment by astorije** [comment]
Sorry @tripu, my git commit is not very clear, you'll see a better explanation in #85.

File: .travis.yml [hunk 1]
```diff
@@ -7,11 +7,3 @@ install:
  - ./.travis-setup.sh
 script:
  - mocha --timeout 20000 --globals Intl,IntlPolyfill,ErrorHandler test/tests.js
-notifications:
-  webhooks:
-    urls:
-      - YOUR_WEBHOOK_URL
```
Comment (astorije) [inline .travis.yml]: @dontcallmedom, @guibbs is not entirely sure this is a wrong copy and paste. I cannot find `YOUR_WEBHOOK_URL` set anywhere so, unless this is set in an environment variable, it looks very much like boilerplate code to me.
Do you confirm?

**Comment by dontcallmedom** [comment]
this comes from @guibbs ' commit https://github.com/w3c/Mobile-Checker/commit/443b6a9238c3b6df52899b80950c727ce9cd8b1e so if he doesn't recognize it, I assume it is indeed copy & paste boilerplate :)

**Comment by astorije** [comment]
Okay @dontcallmedom and @guibbs, good to go then!

**Comment by guibbs** [comment]
yes, indeed! :)

**Pull request merged**